shadow.yaml

```diff
@@ -2,12 +2,13 @@
 package:
   name: shadow
   version: "4.17.3"
-  epoch: 20
+  epoch: 30
```

Comment:
Note to self, check why.

Reply:
Whoops- forgot to include the actual changes! Fixed.